scripts/deploy.ts

```diff
@@ -177,13 +193,73 @@ async function main() {
   )
 
   if (shouldTransferOldAccounts) {
+    const oldKetlAttesationContract = await KetlAttestation__factory.connect(
+      oldKetlAttestationContractAddress,
+      provider
+    )
+
+    const attestationMerkleRoots: { [key: number]: BigNumber } = {}
```

Comment:
Let's extract the type

Reply:
I think it's more helpful to be able to see this inline, and it's not really a standard type that I expect to be reused often (it's only reused because the deploy test code and deploy code is the same).